spec/requests/scim_v2/groups_spec.rb

```diff
@@ -0,0 +1,303 @@
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+#
+# See COPYRIGHT and LICENSE files for more details.
+#++
+
+require "spec_helper"
+
+RSpec.describe "SCIM API Groups" do
+  let(:external_user_id) { "idp_user_id_123asdqwe12345" }
+  let(:external_group_id) { "idp_group_id_123asdqwe12345" }
+  let(:admin) { create(:admin) }
+  let(:oidc_provider) { create(:oidc_provider, slug: "keycloak", creator: admin) }
+  let(:user) { create(:user, identity_url: "#{oidc_provider.slug}:#{external_user_id}") }
+  let(:group) { create(:group, identity_url: "#{oidc_provider.slug}:#{external_group_id}", members: [user]) }
+  let(:headers) { { "CONTENT_TYPE" => "application/scim+json", "HTTP_AUTHORIZATION" => "Bearer access_token" } }
+
+  describe "GET /scim_v2/Groups" do
+    context "with the feature flag enabled", with_flag: { scim_api: true } do
+      before { group }
+
+      it do
+        get "/scim_v2/Groups", {}, headers
+
+        response_body = JSON.parse(last_response.body)
+        expect(response_body).to eq({ "Resources" => [{ "displayName" => group.name,
+                                                        "externalId" => external_group_id,
+                                                        "id" => group.id.to_s,
+                                                        "members" => [{ "value" => user.id.to_s }],
+                                                        "meta" => { "location" => "http://test.host/scim_v2/Groups/#{group.id}",
+                                                                    "created" => group.created_at.iso8601,
+                                                                    "lastModified" => group.updated_at.iso8601,
+                                                                    "resourceType" => "Group" },
+                                                        "schemas" => ["urn:ietf:params:scim:schemas:core:2.0:Group"] }],
+                                      "itemsPerPage" => 100,
+                                      "schemas" => ["urn:ietf:params:scim:api:messages:2.0:ListResponse"],
+                                      "startIndex" => 1,
+                                      "totalResults" => 1 })
+      end
+
+      # it do
+      #   filter = ERB::Util.url_encode('displayName Eq "john"')
+      #   get "/scim_v2/Groups?filter=#{filter}", {}, headers
```

Comment:
:yellow_circle: Should this commented test be reactivated or removed?

Reply:
Yes, it should be reacrtivated.